packages/mockserver-config-writer/src/mockserver-config/package-json.ts

```diff
@@ -58,7 +58,9 @@ function enhanceDependencies(
 function enhanceScripts(fs: Editor, packageJson: Package): void {
     packageJson.scripts ||= {};
     packageJson.scripts['start-mock'] =
-        copyStartScript(packageJson.scripts.start) || `fiori run --config ./ui5-mock.yaml --open \"/\"`;
+        copyStartScript(packageJson.scripts.start) ??
+        packageJson.scripts['start-mock'] ??
```

Comment:
This change ensures that if there is an existing mock-server added by another writer, we dont touch it.